docs/CONTRIBUTION.md

```diff
@@ -0,0 +1,43 @@
+# Contributing to Kubewatch
+
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
```

Comment:
Please un-CamelCase the 2nd part of this sentence, maybe leaving `Pull Requests` only as so.